### DISCUSSION TITLE
Stepper Module scss

### DISCUSSION THREAD

**Discussion by DSyarovIV** [primefaces/community#1588] [PrimeNG]
I have updated from primeng 15 to 17.12.0 
I have created my own theme a while ago so I need to update it in order to use the Stepper component. In the repo for the themes I cannot see the styles for the stepper component. Any suggestions ?

**Comment by cetincakiroglu** [comment]
Hi,

Styles of the stepper generated from primeng-sass-theme. It's located in `theme-base/components/panel/_stepper.scss`
